tools/converter/synapse2ballerina/src/main/java/org/wso2/ei/tools/synapse2ballerina/visitor/SynapseConfigVisitor.java

```diff
@@ -189,12 +193,12 @@ public void visit(Resource resource) {
             logger.debug("Resource");
         }
         org.wso2.ei.tools.converter.common.ballerinahelper.Resource.startResource(ballerinaASTModelBuilder);
-        String[] allowedMethods = {Constant.BLANG_METHOD_GET}; //Default http request method is set to GET
+        String[] allowedMethods = { Constant.BLANG_METHOD_GET }; //Default http request method is set to GET
         if (resource.getMethods() != null) {
-                Map<String, Object> resourceAnnotations = new HashMap<String, Object>();
-                resourceAnnotations.put(Constant.METHOD_NAME, resource.getMethods());
-                Annotation.createResourceAnnotation(ballerinaASTModelBuilder, resourceAnnotations);
-                resourceAnnotationCount++;
+            Map<String, Object> resourceAnnotations = new HashMap<String, Object>();
+            resourceAnnotations.put(Constant.METHOD_NAME, resource.getMethods());
+            Annotation.createResourceAnnotation(ballerinaASTModelBuilder, resourceAnnotations);
```

Comment:
This also can move to the outside of the if-else check, since this used in both check.